ray-operator/test/e2eautoscaler/support.go

```diff
@@ -130,3 +131,11 @@ func workerPodTemplateApplyConfiguration() *corev1ac.PodTemplateSpecApplyConfigu
 						corev1.ResourceMemory: resource.MustParse("1G"),
 					}))))
 }
+
+func getRayClusterWorkerReplicas(cluster *rayv1.RayCluster) int32 {
```

Comment:
Move this function to `ray-operator/test/support/ray.go` and rename it to `GetRayClusterWorkerGroupReplicaSum`

Reply:
Done